source/CommonJobs/CommonJobs.MVC.UI/Areas/Evaluations/Controllers/EvaluationsController.cs

```diff
@@ -0,0 +1,89 @@
+﻿using CommonJobs.Application.Evaluations;
+using CommonJobs.Domain;
+using CommonJobs.Domain.MyMenu;
+using CommonJobs.Infrastructure.Mvc;
+using CommonJobs.Mvc.UI.Infrastructure;
+using NLog;
+using Raven.Client.Linq;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+using System.Web.Mvc;
+using CommonJobs.Utilities;
+using CommonJobs.Domain.Evaluations;
+using CommonJobs.Mvc.UI.Areas.Evaluations.Models;
+using CommonJobs.Application.Evaluations;
+using CommonJobs.Application.EvalForm.Commands;
+
+namespace CommonJobs.Mvc.UI.Areas.Evaluations
+{
+    [CommonJobsAuthorize]
+    //TODO: Add documentation
+    [Documentation("docs/manual-de-usuario/evaluaciones")]
+    public class EvaluationsController : CommonJobsController
+    {
+        private static Logger log = LogManager.GetCurrentClassLogger();
+
+        private string DetectUser()
+        {
+            //TODO: remove hardcoded "CS\\"
+            //TODO: move to an AuthorizeAttribute or something more elegant
+            if (User != null && User.Identity != null && User.Identity.Name != null)
+            {
+                var user = User.Identity.Name;
+                if (user.StartsWith("CS\\"))
+                    user = user.Substring(3);
+                return user;
+            }
+            else
+            {
+                throw new ApplicationException("User cannot be detected");
```

Comment:
RRHH people is working on ensuring to load all employees data in order to avoid that problem. But I am fine on adding a low priority task in order to show a nice help page for this error.